src/rpm/builder.rs

```diff
@@ -307,7 +312,7 @@ impl RPMBuilder {
                     header_digest_sha1.as_str(),
                     header_and_content_digest_md5.as_slice(),
                 )
-                .build(header_and_content_len as i32)
+                .build(header_and_content_len.try_into().unwrap())
```

Comment:
Could you explain why we need to replace the `as` cast? The only advantage I see is a actionable panic rather than a incorrect value, which is great! But then why are we not replacing all of them? https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=c7f6783c4f4b1eb6709102a1f54c7044

Reply:
>actionable panic rather than a incorrect value, which is great! 

That's basically it

>But then why are we not replacing all of them?

I mostly just changed the ones I needed to for this PR, but I think the majority of other instances of casting relate to the constants which are currently `isize`, but they're all manually defined and small numbers so they shouldn't be problematic.

Reply:
I'd prefer if this was done all in one go, or it's only done for the particular item, but iiuc it's neither? Sorry for being picky, I simply want to avoid this project going the same path as upstream.